bindings/kafka/metadata.yaml

```diff
@@ -0,0 +1,238 @@
+# yaml-language-server: $schema=../../component-metadata-schema.json
+schemaVersion: v1
+type: bindings
+name: kafka
+version: v1
+status: stable
+title: "Apache Kafka"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-bindings/kafka/
+binding:
+  output: true
+  input: true
+  operations:
+    - name: create
+      description: "Publish a new message in the topic."
+capabilities: []
+metadata:
+  - name: topics
+    type: string
+    description: |
+      A comma-separated string of topics.
+    example: '"mytopic1,topic2"'
+    binding:
+      input: true
+  - name: brokers
+    type: string
+    required: true
+    description: |
+      A comma-separated list of Kafka brokers.
+    example: '"localhost:9092,dapr-kafka.myapp.svc.cluster.local:9093"'
+    binding:
+      output: true
+      input: true
```

Comment:
When this is the same as the top-level `binding` property, it should be omitted. Including it can lead to errors and confusion